### PR TITLE
Initial commit to notification plugin.

### PR DESCRIPTION
**Pull request by akbhatta**
*Issue #, if available:* N/A

*Description of changes:*
Initial commit to notification plugin with Rest interface just logging debug text.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by anirudha — APPROVED** [review]

File: notification/NOTICE [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Notification
+Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (zhongnansu) [inline notification/NOTICE:2]: change to 2020

**Review by zhongnansu — APPROVED** [review]
Change to use year 2020 in all copyright headers before merging.

File: notification/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#Tue May 19 22:58:14 PDT 2020
```
Comment (davidcui-amzn) [inline notification/gradle/wrapper/gradle-wrapper.properties:1]: Copyright notice?

File: notification/NOTICE (hunk 1 above)
Comment (akbhatta) [reply notification/NOTICE]: fixed

File: notification/gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (akbhatta) [reply notification/gradle/wrapper/gradle-wrapper.properties:17]: fixed

**Pull request merged**